examples/httpd/Cargo.toml

```diff
@@ -7,7 +7,7 @@ publish = false
 
 [dependencies]
 ascii = "1.0"
-tiny_http = { git = "https://github.com/hermitcore/tiny-http.git" }
+tiny_http = "0.11"
```

Comment:
@stlankes, note that this obsoletes our tiny-http fork, since upstream now uses time 0.3 instead of 0.1 which did not work with hermit.